### PR TITLE
Budget App implementation

### PR DESCRIPTION
**Pull request by saluumaa**
In this project I implement:-

1. Category Page 
          - [ ] When the user logs in, they are presented with the categories page.
          - [ ]       For each category, the user can see their name, icon and the total amount of all the transactions that belongs to that category.
          - [ ]       When the user clicks (or taps) on a category item, the application navigates to the transactions page for that category.
          - [ ]       There is a button "add a new category" at the bottom that brings the user to the page to create a new category.
2. Transaction Page
          - [ ] For a given category, the list of transactions is presented, ordered by the most recent.
          - [ ]               At the top of the page the user can see the total amount for the category 
          - [ ]               There is a button "add a new transaction" at the bottom that brings the user to the page to create a new 
                                  transaction.
          - [ ]               When the user clicks on the "Back" button (<), the user navigates to the home page

.
3. New category page
4. New transaction Page

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+> This is a budget app that helps to keep track the expenses of your transactions and it builds using ruby on rails language it shows you every purchases you made and the amount of money you spent on each transaction and it also shows you the total amount of money you spent on all transactions.
+
+## 🛠 Built With <a name="built-with"></a>
+
+<h4> Tech Stack </h4>
+- Ruby on Rails
+- PSQL
+
+## 🛠 live Preview <a name="live-preview"></a>
+<!-- project live preview -->
```
Comment (nedjwab) [inline README.md:42]: the [ ] splash screen should contain a simple page with your name and links to the signup and login pages. Could you fix that as it's part of the requirements?
your splash screen: 👇
splash screen

File: README.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+
```
Comment (nedjwab) [inline README.md:43]: [ ] The live version of your app has some issues, kindly fix that 🤓
error

**Review by nedjwab — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @saluumaa    

Nice job, you are almost there.



## (Highlights) Good Points: :
- Good implementation of some of the project requirements  ✔
- GitHub flow was correctly used ✔.

## (Changes Required) Aspects to improve: ♻️
You did great implementing some of the project requirements, but you still have some issues: 
check the comments below 
 👇👇👇

 
## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 1 above)
Comment (zieeco) [reply README.md:42]: **[REQUIRED CHANGES]**

- Hi :wave: The changes requested by the previous reviewer are related to your [home page](https://github.com/saluumaa/Budget-App/blob/bde34f84c869d304ffb3ab66127ee1d75d217ac5/app/views/home/index.html.erb#L1-L32). According to the design, you should only add the name of your app, link to login and link to signup page. on the contrary, you have added 5 links that don't follow the main design structure. As requested, your design is not yet close to the original design. Please, kindly refer to the main design guideline to update your design to meet the project requirements. this helps you be consistent in following laid down instructions and also, helps you build the skill of being attentive to detail, making a clone of what you see, and not necessarily what you feel like adding. Please, improve on this to fix this issue. :pray: 

*main design*
image

*Your design*
image

**NOTE: The images might be broken as GitHub happens not to be loading images as it ought to. please, refer to the main design, and compare your design to add the required changes**

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @saluumaa,





Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the changes, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- Addressed the deployment issue.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions

- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Whoistolu — APPROVED** [review]
### Hi @saluumaa 

Great job implementing the recommended changes by the previous reviewers 👍🏽 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**